### DISCUSSION TITLE
Turbine bypassratio vs. N2 spool up

### DISCUSSION THREAD

**Discussion by NikolaiVChr** [JSBSim-Team/jsbsim#275] [Q&A]
I looked in the source code, but could not find out how to calculate it.

How do I calculate bypass ratio, if I want 4 seconds at sealevel from 60% (idle) to 100% (mil) N2?

**Comment by legoboyvdlp** [comment]
` N1SpoolUp = new FGSpoolUp(this, BypassRatio, 1.0);`

`N1SpoolDown = new FGSpoolUp(this, BypassRatio, 2.4);`

unless overwritten by custom definition, like in Airbus:

```<function name="N1SpoolUp">
		<table>
			<independentVar lookup="row">/engines/engine[0]/n1</independentVar>
			<tableData>
				 24   1.1
				 28   2.9
				 46   9.7
				 68  15.4
				 80  19.6
				 94  21.4
				103  23.6
			</tableData>
		</table>
	</function>
	
	<function name="N1SpoolDown">
		<table>
			<independentVar lookup="row">/engines/engine[0]/n1</independentVar>
			<tableData>
				 24   1.1
				 28   2.9
				 46   9.7
				 68  15.4
				 80  19.6
				 94  21.4
				103  23.6
			</tableData>
		</table>
	</function>```

**Comment by bcoconni** [comment]
This issue has been moved to "Discussions". Now that this new feature is available in GitHub, it has been decided to move issues related to support requests and questions to "Discussions" and restrict "Issues" to bugs and feature requests.
This change is only motivated by project organization and will not affect the support provided. We'll gladly continue to answer your questions 😄.
